samples/angular/package.json

```diff
@@ -26,14 +26,14 @@
     "jss": "jss",
     "start": "npm-run-all --serial bootstrap:disconnected --parallel start:angular start:proxy start:watch-components",
     "start:connected": "scjss-verify-setup && npm-run-all --serial bootstrap:connected start:angular start:watch-components",
-    "build": "npm-run-all --serial bootstrap:connected --serial build:client build:server --serial build:server-wrapper",
+    "build": "npm-run-all --serial bootstrap:connected --serial build:client build:server",
     "scaffold": "ng generate @sitecore-jss/sitecore-jss-angular-schematics:jss-component",
     "start:angular": "ng serve -o --base-href=/ --source-map=false",
     "start:proxy": "ts-node --project src/tsconfig.webpack-server.json scripts/disconnected-mode-proxy.ts",
     "start:watch-components": "ts-node --project src/tsconfig.webpack-server.json scripts/generate-component-factory.ts --watch",
     "build:client": "cross-env-shell ng build --prod --deploy-url $npm_package_config_sitecoreDistPath/browser/ --output-path=$npm_package_config_buildArtifactsPath/browser/",
     "build:server": "cross-env-shell ng run JssAngularWeb:server:production --output-path=$npm_package_config_buildArtifactsPath/server --output-hashing=none --bundleDependencies true",
-    "build:server-wrapper": "cross-env-shell webpack --config webpack.server.config.js --progress --colors --output-path $npm_package_config_buildArtifactsPath",
+    "postbuild:server": "move-cli ./dist/server/main.js ./dist/server.bundle.js && del-cli ./dist/server",
```

Comment:
Just a thought - as opposed to moving this, I think it should be possible to configure both headless and integrated modes to use the `./dist/server/main.js` output. 

* For [headless](https://jss.sitecore.com/docs/techniques/ssr/headless-mode-ssr), would just set `bundlePath` to './dist/myappname/server/main'.
* For integrated, should be able to set your [app config's](https://jss.sitecore.com/docs/fundamentals/services/app-configuration) `serverBundle` / `filesystemPath` attributes. We could add this by default in the JssAngularWeb.config.

Not sure if there's a clear advantage doing it this way vs moving/renaming the file.

Reply:
@ambrauer By default expected that `server.bundle.js` will be used. We should have consistency between all frameworks
```js
const bundlePath = process.env.SITECORE_JSS_SERVER_BUNDLE || `./dist/${appName}/server.bundle`;
```